cogito/protocol_test.go

```diff
@@ -192,7 +240,8 @@ log_level:             debug
 context_prefix:        the-prefix
 chat_append_summary:   true
 chat_notify_on_states: [success failure]
-sinks: []`
+sinks: []
+github_api_endpoint: `
```

Comment:
I wasn't too sure about what to do here. Most people won't configure the `github_api_endpoint` so the field will be printed as empty. Looks strange, but I needed it to make this test pass.

Reply:
This wouldn't be an issue if you added the default endpoint during JSON parsing:
https://github.com/Pix4D/cogito/blob/6930bc4eb07148496d608fed809ff4762964eccf/cogito/protocol.go#L175
or to be consistent with other fields in `func (src *Source) Validate() error`
https://github.com/Pix4D/cogito/blob/6930bc4eb07148496d608fed809ff4762964eccf/cogito/protocol.go#L271
in if github endpoint is not set; set the default.

I think this is much better approach then sending the some API endpoint from the top (main during putter creation)